build.gradle

```diff
@@ -1947,3 +1947,4 @@ task aggregatedJavadoc(type: Javadoc) {
   else
     options.links "https://docs.oracle.com/javase/8/docs/api/"
 }
+
```

Comment:
Let's remove this extra whitespace change

Reply:
Thanks, that was not intended. Fixed